mod/wikirate/set/right/my_contributions.rb

```diff
@@ -4,17 +4,22 @@ def virtual?
 
 def sections
   @sections ||= begin
-    user_card = left
-    [ {:name=>Card[:metric].name,  :contributions => :contributed_metrics},
-      {:name=>Card[:claim].name,   :contributions => :contributed_claims},
-      {:name=>Card[:source].name,  :contributions => :contributed_sources},
-      {:name=>Card[:wikirate_article].name, :contributions => :contributed_analysis},
-      {:name=>Card[:campaign].name,:contributions => :contributed_campaigns}
-    ].map do |args|
-      c_card     = user_card.fetch(:trait=>args[:contributions])
-      count      = c_card && c_card.contribution_count
-      contr_name = c_card && c_card.cardname.url_key
-      [ (count || 0), args[:name], contr_name ]
+    if left.present?
+      user_card = left
+      [ { name: Card[:metric].name, contributions: :contributed_metrics },
+        { name: Card[:claim].name, contributions: :contributed_claims },
+        { name: Card[:source].name, contributions: :contributed_sources },
+        { 
+          name: Card[:wikirate_article].name, 
+          contributions: :contributed_analysis 
+        },
+        { name: Card[:campaign].name, contributions: :contributed_campaigns }
+      ].map do |args|
+        c_card = user_card.fetch(:trait=>args[:contributions])
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.